src/credentials/FabricTable.h

```diff
@@ -347,6 +348,7 @@ class DLL_EXPORT FabricTableDelegate
 
 private:
     FabricTableDelegate * mNext = nullptr;
+    bool mSelfOwned             = false;
```

Comment:
This is actuall `mOwnedByFabricTable`, right?

Reply:
Renamed